MigrationGuideV8toV10.md

```diff
@@ -0,0 +1,145 @@
+Check log verbosity | `/V:[verbose-log-file]` | `--log-level`
+Specify journal file folder | `/Z:[journal-file-folder]` | Modify environment variable: <br> `AZCOPY_JOB_PLAN_LOCATION`
+Specify parameter file | `/@:<parameter-file>` | Run commands in command line
+Suppress confirmation prompts | `/Y` | Suppressed by default. To enable, specify parameter: INSERT HERE
+Specify number of concurrent operations | `/NC:<number-of-concurrent>` | Modify environment variable: <br> `AZCOPY_CONCURRENCY_VALUE`
+Specify source/destination type | `/SourceType:<option>` `/DestType:<option>` Options: blob, file | `--from-to=[enums]` <br> (typically not used)
+Upload contents recursively | `/S` | `--recursive`
+Match a specific pattern | `/Pattern:<pattern>` | `--include-pattern string` <br> `--exclude-pattern string` <br> `--include-path string` <br> `--exclude-path string`
+Create an MD5 hash when downloading data | Always does this | `--put-md5`
+Check the MD5 hash when downloading data | `/CheckMD5` | `--check-md5=[option]` <br> Options: NoCheck, LogOnly, FailIfDifferent (default), FailIfDifferentOrMissing
+Retrieve listing | `/L` | `azcopy list`
+Set modified time to be same as the source blobs | `/MT` | `--preserve-last-modified-time`
+Exclude newer source | `/XN` | Not yet supported
+Exclude older source | `/XO` | Use the sync command
+Upload archive files/blobs | `/A` | See row below
+Set attributes | `/IA:[RASHCNETOI]` <br> `/XA:[RASHCNETOI]` | `--include-attributes string` <br> `--exclude-attributes string`
+Copy blobs or files synchronously among two Azure Storage endpoints | `/SyncCopy` | V10 is always synchronous from source to destination, unlike v8 which downloads then re-uploads
+Set content type | `/SetContentType:[content-type]` | `--content-type string`
+Set blob type at destination | `/BlobType:<option>` Options: page, block, append | `--blob-type string`
+Use specified block size | `/BlockSizeInMb:<block-size-in-mb>` | `--block-size-mb float`
+Set file name delimiters | `/Delimiter:<delimiter>` | N/A
+Transfer blob snapshots | `/Snapshot` | Coming soon
+
+## Common Questions
+
+### How does service-to-service transfer differ in V8 and V10?
+V8 either schedules transfers (async) or downloads the data and re-uploads it (streaming).
+V10 uses new synchronous copy APIs where data is read on the server side.
+
+### How can I use AzCopy to work with Azure Tables?
+The latest version that supports tables is AzCopy 7.3.
+
+### How can I use AzCopy to work with Azure Queues?
+The latest version that supports queues is AzCopy 8.1.
+
+### What is different between the job management (how to resume jobs in both use examples)?
+In v8, you can rerun the same command from before and answer the prompt.
+In v10, you can have a job sub group where you can resume with job id.
```

Comment:
Maybe link to: https://docs.microsoft.com/en-us/azure/storage/common/storage-ref-azcopy-jobs-resume

Reply:
Done